client/app/components/menu_refresh_indicator.coffee

```diff
@@ -0,0 +1,68 @@
+{span, button, img} = React.DOM
+
+LayoutActionCreator  = require '../actions/layout_action_creator'
+
+{Spinner} = require './basic_components'
+
+module.exports = React.createClass
```

Comment:
I think a comment about what the component does could be interesting. What do you think about it?